package.json

```diff
@@ -47,7 +47,7 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"@samply/lens": "0.4.4-0",
+		"@samply/lens": "0.4.5",
```

Comment:
Consider updating to 0.4.6-0, that already should contain the fix for the empty lens query

Reply:
Yes, I would also like this merged: https://github.com/samply/bbmri-sample-locator/pull/21
But it is yours and still draft, so it's your decision. 
(I can't merge in this repo)